commercetools/resource_product_discount_test.go

```diff
@@ -0,0 +1,213 @@
+package commercetools
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccProductDiscountCreate_basic(t *testing.T) {
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckProductDiscountDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccProductDiscountConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "key", "standard",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "name.en", "standard name",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "description.en", "Standard description",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "sort_order", "0.95",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "predicate", "1=1",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "valid_from", "2018-01-02T15:04:05Z",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "valid_until", "2019-01-02T15:04:05Z",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "value.0.type", "relative",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "value.0.permyriad", "1000",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_product_discount.standard", "is_active", "true",
+					),
```

Comment:
Let's add a check as in 
 - https://github.com/labd/terraform-provider-commercetools/pull/265/files#diff-f01e4c64f1877c4a0abc1852295c2046f5a73d2eaac80b17297147734a066aa3R340
 -  https://github.com/labd/terraform-provider-commercetools/pull/265/files#diff-f01e4c64f1877c4a0abc1852295c2046f5a73d2eaac80b17297147734a066aa3R164